app/src/main/java/com/example/myfridgehome/ui/main/MainViewModel.kt

```diff
@@ -14,6 +14,10 @@ class MainViewModel : ViewModel() {
     }
 
     fun fetchRecipes() {
-        _recipes = recipeService.collectRecipes()
+        _recipes = recipeService.fetchRecipes()
     }
+
+    var recipes:MutableLiveData<ArrayList<Recipe>>
```

Comment:
The variable _recipes is already public.  Adding a getter/setter makes sense in combination with a private variable.  So, if doing this, make _recipes private as well.